### ISSUE TITLE
 Failed sysctl call: hw.cpufrequency

### ISSUE THREAD

**Issue by entity** [PaperSpigot/Paper#10067]
### Timings or Profile link

N/A

### Description of issue

When running a fresh version of Paper (no plugins) on either 1.20.2 or even 1.20.4 I get an error regarding `hw.cpufrequency`.

```
charlie@Charlies-MBP MCServer % java -jar paper-1.20.2-318.jar -nogui
Starting org.bukkit.craftbukkit.Main
System Info: Java 21 (OpenJDK 64-Bit Server VM 21.0.1+12-LTS) Host: Mac OS X 14.2.1 (aarch64)
Loading libraries, please wait...
[14:55:37 WARN]: [oshi.util.platform.mac.SysctlUtil] Failed sysctl call: hw.cpufrequency, Error code: 2
```

### Plugin and Datapack List

N/A - Fresh Install

### Server config files

N/A - Fresh Install

### Paper version

> ver
[14:55:47 INFO]: Checking version, please wait...
[14:55:47 INFO]: This server is running Paper version git-Paper-318 (MC: 1.20.2) (Implementing API version 1.20.2-R0.1-SNAPSHOT) (Git: 9271ee7)
You are running the latest version

### Other

I am running an Apple Silicon chip, precisely the M3 Max.

**Comment by MiniDigger** [comment]
nothing we can do, and this is only cosmetic, right? (as in server still starts?)
please report it to oshi, maybe the maintainer can provide a hint on what data they need to resolve this

**Comment by heychazza** [comment]
> nothing we can do, and this is only cosmetic, right? (as in server still starts?) please report it to oshi, maybe the maintainer can provide a hint on what data they need to resolve this
> 
> edit: or maybe you can create a minimal repro, mojang only includes oshi 6.4.5, latest is 6.4.9, maybe its already fixed? something simple like `fun main() = println(oshi.SystemInfo().hardware.processor)` should trigger it https://github.com/oshi/oshi

Hey buddy,

Thanks for the quick reply! Yup the server still starts, I just wasn’t sure if any multi-threading applied and this error would prevent anything.

Granted, it’s only a development server but still.

I’ll look at seeing about getting a repro sorted :)

**Comment by MiniDigger** [comment]
preeeeeetttty sure mojang only uses oshi for crash report info
but yeah, the maintainer will appreciate it, I will close this issue then

**Comment by electronicboy** [comment]
https://github.com/oshi/oshi/issues/2547

**Comment by dbwiddis** [comment]
> please report it to oshi, maybe the maintainer can provide a hint on what data they need to resolve this

OSHI maintainer here.  Would love someone with an M3 to help provide some info form their ioregistry at the issue in the above comment.  I have a guess at a fix.

**Comment by heychazza** [comment]
> > please report it to oshi, maybe the maintainer can provide a hint on what data they need to resolve this
> 
> OSHI maintainer here. Would love someone with an M3 to help provide some info form their ioregistry at the issue in the above comment. I have a guess at a fix.

Happy to help! I’ve got an M3 Max, what do I need to run?

**Comment by dbwiddis** [comment]
See this comment: https://github.com/oshi/oshi/issues/2547#issuecomment-1868338969

**Comment by dbwiddis** [comment]
> oshi 6.4.5, latest is 6.4.9, maybe its already fixed?

To close the loop if anyone else finds this issue in search, this was fixed in OSHI 6.4.10, released a few minutes ago.
